akvo/rsr/views/utils.py

```diff
@@ -45,13 +45,13 @@ def check_project_viewing_permissions(request, project):
 def get_hierarchy_grid(project, include_private=False):
     """Return a hierarchy grid for a project"""
 
-    parents = project.parents() if not include_private else project.parents_all()
-    siblings = project.siblings() if not include_private else project.siblings_all()
-    children = project.children() if not include_private else project.children_all()
+    ancestors = project.ancestors().published().public() if not include_private else project.ancestors()
+    siblings = project.siblings().published().public() if not include_private else project.siblings()
+    children = project.children().published().public() if not include_private else project.children()
```

Comment:
Not sure about this. `parents_all` might not mean ancestors

Reply:
I'm not even sure if this is used anymore. I can't see any reference to it in the frontend.

Reply:
There really isn't any equivalent query to get ancestors in the previous codebase, `.parents()` and `.parents_all()` are because the data model allows returning multiple objects even if they are not logically correct. the equivalent query should be `.parent()`

Reply:
Fixed in 0bdf7a0377119604493b898af3b1b212d54eaee5